lib/helpers.js

```diff
@@ -0,0 +1,83 @@
+var fs = require('fs-extra');
+var chalk = require('chalk');
+
+function helpers () {
+  function extractCompilerData (path) {
+    var res = {};
+    var match = /v\d\.\d\.\d/.exec(path);
```

Comment:
Regex were not commented in the past I think and this is really basic. If /v\d\.\d\.\d/ is not clear, people will probably not get to the regex anyway :) If more folks want comments there, please mention it, I also don´t mind.